lib/usd/ui/USDImportDialog.cpp

```diff
@@ -172,4 +173,13 @@ void USDImportDialog::onItemClicked(const QModelIndex& index)
 	}
 }
 
+void USDImportDialog::onResetFileTriggered()
+{
+	if (fTreeModel)
```

Comment:
What is our coding standard stance on if checks? In Maya only a bool is a boolean, so if the type is not a boolean don't use it as one.

Reply:
Not sure if we've listed this in the coding standard or not, but I can add a "nullptr !=" here.